shell2http.go

```diff
@@ -294,8 +294,11 @@ func getShellHandler(appConfig Config, path string, shell string, params []strin
 			shellOut, err = osExecCommand.Output()
 		}
 
-		if err != nil && !appConfig.showErrors {
+		if err != nil {
 			log.Println("exec error: ", err)
+		}
+
+		if err != nil && !appConfig.showErrors {
```

Comment:
By this logic, the output will be mixed with error line. For example,

    exec error: ....
    <html>....

It's not something that would be expected. As a variant to add a third option:), in which all (standard output + error line) will be displayed. But it seems it will be a complication. I wanted to keep it simple.